source/includes/code-examples/insert-many/InsertMany.cs

```diff
@@ -49,8 +49,11 @@ private static void InsertManyRestaurants()
         // Generates 5 new restaurants by using a helper method
         var restaurants = GenerateDocuments();
 
-        // Inserts the new documents into the restaurants collection
-        _restaurantsCollection.InsertMany(restaurants);
+        // Creates an option object to bypass documentation validation on a the documents
+        var options = new InsertManyOptions() { BypassDocumentValidation = true };
+
+        // Inserts the new documents into the restaurants collection with the specified options
+        _restaurantsCollection.InsertMany(restaurants, options);
```

Comment:
I think the method with the `options` added should be a separate block of code in this file, rather than adding the options line into this code block. This code is also used on the [Usage Examples page](https://docs-mongodbcom-staging.corp.mongodb.com/csharp/docsworker-xlarge/delete-outdated-insert/usage-examples/insertMany/) (see the Synchronous tab on the first code block), so this change is showing a different variation of the code for sync than what shows for async. We also don't want to show the options on the usage example page since they're not relevant for that example. 

Happy to discuss this more if needed!

Reply:
Should have caught that one.